assets/javascripts/discourse/components/reviewable-post-voting-comment.js

```diff
@@ -0,0 +1,15 @@
+import Component from "@glimmer/component";
+import { tracked } from "@glimmer/tracking";
+import { inject as service } from "@ember/service";
+
+export default class ReviewableChatMessage extends Component {
+  @service store;
+  @tracked post;
+
+  constructor() {
+    super(...arguments);
+    this.store.find("post", this.args.reviewable.post_id).then((post) => {
+      this.post = post;
+    });
+  }
```

Comment:
I think we could do something like

```suggestion
  constructor() {
    super(...arguments);
    this.fetchPost();
  }
  
  async fetchPost() {
     this.post = await this.store.find("post", this.args.reviewable.post_id);
  }
```